tools/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Shell script equivalent of Metamath BUILD tool
+# Mario Carneiro 7-Jan-2022
+
+usage() {
+  cat << HELP
+BUILD - Build a file with multiple tokens per line from a list
+
+This command combines a list of tokens into multiple tokens per line,
+as many as will fit per line, separating them with spaces.
+Syntax:  build.sh < INFILE > OUTFILE
```

Comment:
Suggest to add: A line is terminated after it became more than 72 characters wide.

Reply:
It's actually 73 characters, because of the space insertion. I think this has to do with the recommended line length in git commit messages, but I can't say for sure.

Reply:
ack